giskard-frontend/src/components/ResultPopover.vue

```diff
@@ -0,0 +1,230 @@
+                                        </div> 
+                                        <div v-else> - </div>
+                                    </div>
+                                </template>
+                                <span class="predict"> {{actual}}</span>
+                            </v-tooltip>
+
+                        </v-col>
+                    </v-row>
+                    <v-row align="center">
+                        <v-col cols="4">
+                            <v-text-field
+                                dense
+                                solo
+                                hide-details
+                                clearable
+                                class="mx-2 mb-2 flex-1"
+                                v-model="search"
+                                append-icon="mdi-magnify"
+                                label="Search"
+                            ></v-text-field>
+                        </v-col>
+                        <span>
+                            ({{ numberDisplayed }} of {{ totalCategories }})
+                        </span>
+                    </v-row>
+                    <v-chart class="chart pr-5" :option="chartOptions" :init-options="chartInit" autoresize/>
+                </v-card-text>
+                <v-card-actions class="justify-end">
+                    <v-btn
+                    text
+                    @click="dialog.value = false"
+                    >Close</v-btn>
+                </v-card-actions>
+            </v-card>
+        </template>
+    </v-dialog>
+</template>
+
+<script lang="ts">
+import * as _ from "lodash";
+import {Component, Prop, Vue, Watch} from 'vue-property-decorator';
+import {use} from "echarts/core";
+import ECharts from "vue-echarts";
+import {SVGRenderer} from "echarts/renderers";
+import {BarChart} from "echarts/charts";
+import {GridComponent} from "echarts/components";
+import {DataZoomSliderComponent} from "echarts/components"
+import {DataZoomInsideComponent} from "echarts/components"
+import {sliceStr} from "@/utils";
+
+use([SVGRenderer, BarChart, GridComponent, DataZoomSliderComponent, DataZoomInsideComponent]);
+Vue.component("v-chart", ECharts);
+
+@Component
+export default class ResultPopover extends Vue {
+    @Prop({required: true}) resultProbabilities!: {[key: string]: number};
+    @Prop({required: true}) prediction!: string;
+    @Prop({required: true}) actual!: string;
+    @Prop({required: true}) classColorPrediction!: string;
+
+    search: string = "";
+    showSlider : boolean = true;
+    numberDisplayed : number = 0;
+    windowWidth = window.innerWidth;
+
+    sliceStr(s, n, m){
+        return sliceStr(s, n, m);
+    }
+    async mounted() {  
+        window.addEventListener('resize', () => {
+        this.windowWidth = window.innerWidth
+        })
+  }
+
+    get totalCategories(){
+        return Object.keys(this.resultProbabilities).length;
+    }
+
+    get maxCharsCategory(){
```

Comment:
same comment as in PredictionResults.vue, plus instead of copying the code we should import it from a shared ts file